src/options.js

```diff
@@ -0,0 +1,22 @@
+export const positions = {
+  TOP_LEFT: 'top left',
+  TOP_CENTER: 'top center',
+  TOP_RIGHT: 'top right',
+  MIDDLE_LEFT: 'middle left',
+  MIDDLE: 'middle',
```

Comment:
`middle center`, maybe?

Reply:
I was afraid of tautology. Do not know the best way to name it here

Reply:
Yeah, I see you point, but I guess if we look at it like a string that actually represents the vertical position with the first word and the horizontal position in the second one, it makes sense.

Maybe we could accept an object as the position option, like `{ vertical: 'center', horizontal: 'center' }` then we could even just use center instead of middle...but it would be a break change, unless we accept both forms, the string and the object, but push for the latter in the docs, since the string would be weird like `center center`.

What you thing?

Reply:
In a second thought, I think it's not worth the work to accept objects haha 

With the addition of your built-in constant for transitions it's more than fine to go with only `MIDDLE`.

LGTM and update the docs later.

Reply:
Thank you!